src/controllers/userconroller.js

```diff
@@ -0,0 +1,91 @@
+import nodemailer from 'nodemailer';
+import db from '../database/models/index';
+import catchAsync from '../utils/catchAsync';
+
+const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: 'charlesntwari2@gmail.com',
+    pass: 'orvilqiccnltvonu',
```

Comment:
Please put this inside the .env file. your email password should not be exposed like this